src/main/java/com/notably/logic/suggestion/SuggestionEngineImpl.java

```diff
@@ -61,56 +55,54 @@ public void suggest(String userInput) {
-            return new ErrorSuggestionCommand(
-                    "Invalid command format. To see the list of available commands, type: help");
+            model.setResponseText("\"" + userInput + "\" is an invalid command format. "
+                    + "To see the list of available commands, type: help");
+            return Optional.empty();
         }
 
         String commandWord = matcher.group("commandWord");
         CorrectionResult<String> correctionResult = commandCorrectionEngine.correct(commandWord);
         if (correctionResult.getCorrectionStatus() == CorrectionStatus.FAILED) {
-            return new ErrorSuggestionCommand(
-                    "Invalid command. To see the list of available commands, type: help");
+            model.setResponseText("\"" + userInput + "\" is an invalid command format. "
+                    + "To see the list of available commands, type: help");
+            return Optional.empty();
         }
         commandWord = correctionResult.getCorrectedItems().get(0);
 
         final String arguments = matcher.group("arguments");
 
         switch (commandWord) {
-        case OpenSuggestionCommand.COMMAND_WORD:
-            try {
-                return new OpenSuggestionCommandParser(model, pathCorrectionEngine).parse(arguments);
-            } catch (ParseException e) {
-                return new ErrorSuggestionCommand(e.getMessage());
-            }
-
-        case DeleteSuggestionCommand.COMMAND_WORD:
-            try {
-                return new DeleteSuggestionCommandParser(model, pathCorrectionEngine).parse(arguments);
-            } catch (ParseException e) {
-                return new ErrorSuggestionCommand(e.getMessage());
-            }
+        case "open":
+            return new OpenSuggestionCommandParser(model, pathCorrectionEngine).parse(arguments);
+
+        case "delete":
+            return new DeleteSuggestionCommandParser(model, pathCorrectionEngine).parse(arguments);
 
         /*case SearchSuggestionCommand.COMMAND_WORD:
             return new SearchSuggestionCommandParser(model).parse(arguments);*/
 
-        case NewSuggestionCommand.COMMAND_WORD:
-            return new NewSuggestionCommand();
+        case "new":
+            return new NewSuggestionCommandParser(model).parse(arguments);
 
-        case EditSuggestionCommand.COMMAND_WORD:
-            return new EditSuggestionCommand();
+        case "edit":
+            model.setResponseText("Edit this note");
+            return Optional.empty();
 
-        case HelpSuggestionCommand.COMMAND_WORD:
-            return new HelpSuggestionCommand();
+        case "help":
+            model.setResponseText("Display a list of available commands");
+            return Optional.empty();
 
-        case ExitSuggestionCommand.COMMAND_WORD:
-            return new ExitSuggestionCommand();
+        case "exit":
+            model.setResponseText("Exit the application");
+            return Optional.empty();
```

Comment:
I'd say we create different `...SuggestionCommandParser` for each of these types, to make the design more consistent!